orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/pipeline/ManualJudgmentStage.groovy

```diff
@@ -73,20 +81,53 @@ class ManualJudgmentStage implements StageDefinitionBuilder, AuthenticatedStage
     final long timeout = TimeUnit.DAYS.toMillis(3)
 
     @Autowired(required = false)
-    EchoService echoService
+    private final EchoService echoService
+
+    @Autowired(required = false)
```

Comment:
This is still not correct.

The `@Autowired` should be on the constructor and not these attributes.

Reply:
I removed, but i find the @Autowired in many places. This should not be a concern.